pom.xml

```diff
@@ -147,10 +147,16 @@
                 <artifactId>slf4j-log4j12</artifactId>
                 <version>${slf4j.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.github.java-diff-utils</groupId>
+                <artifactId>java-diff-utils-parent</artifactId>
+                <type>pom</type>
+                <version>4.7</version>
+            </dependency>
             <dependency>
                 <groupId>io.github.java-diff-utils</groupId>
                 <artifactId>java-diff-utils</artifactId>
-                <version>4.8</version>
+                <version>4.7</version>
```

Comment:
I resolved the issue, do not see the problem

Reply:
Yes, thanks